src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -307,6 +307,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     val fileContent =
       s"""
          |#!/bin/bash
+         |export _JAVA_OPTIONS=-Djava.io.tmpdir=$JesCromwellRoot/tmp
```

Comment:
Any reason we might want/not want to include whatever the current `_JAVA_OPTIONS` value is?

```
export _JAVA_OPTIONS="$_JAVA_OPTIONS -Djava.io.tmpdir=$JesCromwellRoot/tmp"
```